src/fast_forward.rs

```diff
@@ -213,8 +215,8 @@ mod tests {
         let new_parents_parent = hex::decode(new_parents_parent).unwrap();
 
         let mut a = Allocator::new_limited(500000000, 62500000, 62500000);
-        let puzzle = spend.puzzle_reveal.to_clvm(&mut a).expect("to_clvm");
-        let solution = spend.solution.to_clvm(&mut a).expect("to_clvm");
+        let puzzle = node_from_bytes(&mut a, spend.puzzle_reveal.as_slice()).expect("to_clvm");
+        let solution = node_from_bytes(&mut a, spend.solution.as_slice()).expect("to_clvm");
```

Comment:
I think we should preserve the ability for `Program` to convert to and from `NodePtr`

Reply:
Yeah, I agree - I'll make sure this is included